pom.xml

```diff
@@ -1230,6 +1230,12 @@
                 <version>${spring.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-context-support</artifactId>
+                <version>${spring.version}</version>
+            </dependency>
```

Comment:
@MarieVerdonck : In that case, I think this can be safely removed from this PR.  It's not necessary on `main` anymore, so there's no need to re-add it in this PR.